docs/proposals/milestone-1-defaults.md

```diff
@@ -0,0 +1,32 @@
+# Milestone-1 defaults
+
+### Terminology
+Proposed terminology, one of these two terms need to replace `--hub-router`, either `--discovery-node`, 
+`--relay-node` (or something else):
+
+- _Relay Node_: is for hard NAT traversal (symmetric NAT only) The hub in a hub/spoke model.
+- _Discovery Node_: ICE/NAT traversal discovery.
+
+### Default Deployments
+
+1. The SaaS demo will NOT provide a discovery or relay node. While the functionality is there, 
+the implementation relies on Wireguard state from the Discovery Node (e.g. hub-router). 
+A user can add their own discovery/relay node offered via an image or starting the agent as that node. 
+Alternatively, they can kick the tires using option #2 in this list.In the long run, NAT-T discovery 
+is a vital component to the user experience and should be included in an organization be default. 
+The mechanism to get there will be generic, Anil is currently on point there but the more eyes/ideas 
+the better there. It is not a trivial task and will need extensive testing in diverse environments 
+that we will all need to help vet.In the short term using Wireguard state works very well and can be 
+leveraged for demo environments to accomplish ICE like functionality. Making the discovery process 
+driver agnostic (to facilitate future drivers such as QUIC) and not a peer in the mesh is the goal. 
+Discovery node being provisioned from the stack per/organization will be a milestone target in the future.
```

Comment:
This paragraph includes two things:

1. What we plan to provide in the SaaS Preview deployment.
2. Thoughts about what may happen in the future.

It would be great to separate these out. Cover first what is proposed for the SaaS Preview and the reasoning behind it. Have another section later that discusses "Future Work".